### PR TITLE
Retrieve the PGPASSWORD from env var, not from secrets mount

### PR DESCRIPTION
**Pull request by tlrh314**

### CODE REVIEW

File: src/dso_api/settings.py [hunk 1]
```diff
@@ -142,20 +142,17 @@
...
-                "HOST": env.str("PGHOST"),
-                "PORT": env.str("PGPORT"),
-                "OPTIONS": {"sslmode": "require"},
-            }
+    DATABASES = {
+        "default": {
+            "ENGINE": "django.contrib.gis.db.backends.postgis",
+            "NAME": env.str("PGDATABASE"),
+            "USER": env.str("PGUSER"),
+            "PASSWORD": env.str("PGPASSWORD"),
```
Comment (jjmurre) [inline src/dso_api/settings.py:150]: Could you also describe the "why" of the PR in the commit message.

File: src/dso_api/settings.py [hunk 2]
```diff
@@ -142,20 +142,17 @@
 }
 
 if CLOUD_ENV.lower().startswith("azure"):
-    with open("/mnt/secrets-store/mdbdataservices-read") as secrets_file:
```
Comment (jjmurre) [inline src/dso_api/settings.py:145]: Is it possible to trace back from the commit messages why this was done this way?

Comment (tlrh314) [reply src/dso_api/settings.py:145]: I've updated the commit message to be more verbose on the why.

File: src/dso_api/settings.py (hunk 1 above)
Comment (tlrh314) [reply src/dso_api/settings.py:150]: I've updated the commit message to be more verbose on the why.

File: src/dso_api/settings.py (hunk 2 above)
Comment (tlrh314) [reply src/dso_api/settings.py:145]: The current implementation comes from https://github.com/Amsterdam/dso-api/pull/355, which probably overlooked that the env var `PGPASSWORD` is exported in the k8s manifest in addition to volume mounting it.

**Review by lbam — APPROVED** [review]

**Pull request merged**